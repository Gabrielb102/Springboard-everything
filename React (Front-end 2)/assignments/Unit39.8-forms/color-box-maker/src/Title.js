const Title = ({length}) => {
    if (length === 1) {
        return <h1>Our Happy Box</h1>;
    } else if (length > 1) {
        return <h1>Our Big Happy Box Family ☺️</h1>;
    } else {
        return <h1>No Boxes</h1>
    }
}

export default Title;