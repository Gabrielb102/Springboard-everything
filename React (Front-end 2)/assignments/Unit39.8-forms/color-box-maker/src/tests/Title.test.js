import { render, fireEvent } from "@testing-library/react";
import Title from "../Title";

describe("Tests for Title Component", () => {

    it("Renders", () => {
        render(<Title/>);
    })
    it("Renders a consistent user interface", () => {
        const id = '3516516548'
        const { asFragment } = render(<Title id={id} color='red' height='186px' width='36px' removeFunc={() => removeFunc(id)}/>);
        expect(asFragment).toMatchSnapshot();
    })
})