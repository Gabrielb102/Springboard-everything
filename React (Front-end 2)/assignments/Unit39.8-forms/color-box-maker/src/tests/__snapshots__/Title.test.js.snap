// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for Title Component Renders a consistent user interface 1`] = `[Function]`;
